CryptoHeroez

```diff
@@ -0,0 +1,111 @@
+{
+    "project": "Crypto Heroez",
+    "tags": [
+        "Dorian Nakamoto",
+        "DorianNakamoto",
+        "Nick Szabo",
+        "NickSzabo",
+        "Hal Finney",
+        "HalFinney",
+        "Vitalik Buterin",
```

Comment:
Remove all names from tags, they're too vague / common and will mark any nft with the name "Charles Hoskinson" as a fake.

Reply:
Ok, I can.. but how can you spot fakes when I remove them? I have a proposal - what if you search tags also in metadata (basically from the value you have in "tags" box in nft detail? I have always author or website there and it's very specific for the project.

Reply:
Also if you have vague asset name like "Charles Hoskinson", you can match multiple projects. And NFT would be valid when you find at least one valid policy for that tag. Only issue it could cause is when there is a project with the same asset name that is not registered yet. But they can always register their set of policies and everything should be OK, right?

Reply:
yeah sure, but it will say everything that isn't verified and has "charles hoskinson" "is not a real crypto heroez", it would be sufficient to have a checkmark on your assets and not label other charles hoskinsons nfts as copies.